spec/requests/court_dates_spec.rb

```diff
@@ -63,63 +64,7 @@
 
         document_inspector = DocxInspector.new(docx_contents: response.body)
 
-        expect(document_inspector.word_list_document_contains?(court_date.date.to_s)).to eq(true)
```

Comment:
deleted tests? :(

Reply:
The new template doesn't have a judge name field or a hearing type field. My understanding was that these fields are no longer wanted and so I removed the tests that checked for them. Would it be better to keep a test the ensured they weren't there anymore?

Reply:
Court dates should definitely have judge and hearing type, so something is wrong here...

Reply:
I must have misunderstood the request then. I thought they wanted to use an existing document. What you're say is that we want to make a new document that's themed like an existing document but with different data. I'll talk with the issue holder to clarify.